taxcalc/current_law_policy.json

```diff
@@ -3679,7 +3679,6 @@
         "section_1": "Nonrefundable Credits",
         "section_2": "Child Tax Credit",
         "irs_ref": "",
-        "notes": "The Tax Cuts and Jobs Act sets this at $600 for joint and $300 for others",
```

Comment:
@MaxGhenis,  Can't remove the "notes", but can set to this: `"notes": "",`

Reply:
Oops, fixed.

Reply:
Thanks @MaxGhenis.  I've been pretty much out of the loop on the details of specifying the iterations of the TCJA reforms, so I think it is best to wait for Cody's review.  Thanks for your careful review of our work.